apps/api/src/app/scheduler/scheduler.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from "@nestjs/common";
+import { Cron } from '@nestjs/schedule';
+import { RedisLockService } from '@huangang/nestjs-simple-redis-lock';
+import { AppService } from "../app.service";
+
+@Injectable()
+export class SchedulerService {
+    constructor(
+        protected readonly lockService: RedisLockService,
+        protected readonly appService: AppService){}
+
+    @Cron(process.env.CLICK_BACKUP_CRON)
```

Comment:
`ConfigServerice` everywhere. We may not deploy by putting the config in the env in some use cases, this will fail then.